tests/unit/authenticators/jwt-test.js

```diff
@@ -71,7 +72,7 @@ test('#restore resolves when the data includes `token` and `expiresAt`', functio
 
   var data = {};
   data[jwt.tokenPropertyName] = token;
-  data[jwt.tokenExpireName] = expiresAt;
+  data.expiresAt = expiresAt;
```

Comment:
since the method never looks for `data[jwt.tokenExpireName]` this is equivalent to not setting the value so assumed it to be an error, if it just shouldn't be there I can update this